client/src/document/ingredients/browser-compatibility-table/utils.ts

```diff
@@ -191,3 +191,20 @@ export function getCurrentSupport(
   // Default (likely never reached)
   return getFirst(support);
 }
+
+export function getPreviousVersion(
+  version: BCD.VersionValue | string | undefined,
+  browser: BCD.BrowserStatement
+): BCD.VersionValue | string | undefined {
+  if (browser && typeof version === "string") {
+    const browserVersions = Object.keys(browser["releases"]).sort(
+      (a, b) => Number(a) - Number(b)
```

Comment:
Browser versions can't be correctly sorted as numbers, you'll need something like the npm compare-versions package.

Reply:
Oh yeah.  The browser info I had looked at could be interpreted as decimals safely. But I see deno and Node JS can not be (deno has 1.10 after 1.9 and Node JS has 0.10.0 etc.

I'm busy for the rest of the day but I'll look into this tomorrow.

Reply:
Added compare-versions and using it now. The new "After" picture shows 1-4.4.3 instead of 1-4.4 for Webview Android, which is the correct range.